### PR TITLE
Begin converting the README to a timeline

### PR DESCRIPTION
**Pull request by gabebw**
@mxie @janetriley @galiat @railsbridge-boston/owners 

I've converted the README to a timeline with each person's responsibilities at any given point from 6 months before up to Installfest. I'd love your feedback on whether this timeline makes sense, or what I've left out or not explicitly mentioned in the months and weeks leading up to an event.

### CODE REVIEW

**Comment by julissa** [comment]
There should be a timeline event for the sponsorship coordinator. They should start emailing companies 8 weeks before the workshop.

**Comment by gabebw** [comment]
We should mention retros after the event.

File: README.md [hunk 1]
```diff
@@ -1,56 +1,148 @@
...
 
-Students and TAs can apply to the workshop from [the RailsBridge Boston
-homepage](http://www.railsbridgeboston.org/). The signup link is removed when
-registration is closed.
+* Event title: Railsbridge Boston
+* Event audience: Women in Boston
+* Registration link: Eventbrite link
+* Twitter handle: railsbridgebos
+* Event description: Railsbridge Boston (http://www.railsbridgeboston.org/) is a
+  two-day workshop that teaches women in the Boston community Ruby on Rails.
```
Comment (mxie) [inline README.md]: How about "teaches Ruby on Rails to women in the Boston community"?

File: README.md [hunk 2]
```diff
@@ -1,56 +1,148 @@
...
+[nerd-form]: http://microsoftcambridge.com/eventmanager/
 
-## Signup links
+They will reply with a confirmation email that includes a link to an event
+details form. Here are some things you can use to fill out that form:
 
-Students and TAs can apply to the workshop from [the RailsBridge Boston
-homepage](http://www.railsbridgeboston.org/). The signup link is removed when
-registration is closed.
+* Event title: Railsbridge Boston
```
Comment (mxie) [inline README.md]: `RailsBridge`

File: README.md [hunk 3]
```diff
@@ -1,56 +1,148 @@
...
+* Event audience: Women in Boston
+* Registration link: Eventbrite link
+* Twitter handle: railsbridgebos
+* Event description: Railsbridge Boston (http://www.railsbridgeboston.org/) is a
+  two-day workshop that teaches women in the Boston community Ruby on Rails.
+* Event logo: Use [this file](/logos/logo.png).
```
Comment (mxie) [inline README.md]: Errr...maybe this instead?

railsbridge-train-black

File: README.md [hunk 4]
```diff
@@ -1,56 +1,148 @@
...
 
-## Current Workshop
+## Constantly (Organizer)
 
-[April 2015](./coordinating-2015-april.md)
+These tasks are not around organizing a specific event - the organizer should
+check in with them reasonably often.
 
-## Workshop Task Descriptions
+Look in our [private repo][private] for credentials for our GMail/twitter/etc.
```
Comment (mxie) [inline README.md]: `Twitter`

File: README.md [hunk 5]
```diff
@@ -1,56 +1,148 @@
...
+## 1.5 weeks before (Catering)
 
-We keep [an announcement list][list] for people who want to know about upcoming
-workshops. If someone can't make it to this one, direct them there.
+We place catering orders ~1.5 weeks before the workshop. We assume about 100
+people will attend, but double check that number. For more information, see
+[this document](/catering.md).
 
-[list]: http://railsbridgeboston.us6.list-manage.com/subscribe?u=1b4272afae4569dec6efb74bb&id=1ec91857a1
+## 4 weeks before (Organizer)
```
Comment (mxie) [inline README.md]: I don't think this is Organizer-specific. It'd be whoever had set up the registration who will make it live. The organizer can probably do the announcing though.

**Comment by mxie** [comment]
Anything related to the workshop venue and maybe even the after-party venue should be handled by a Venue Coordinator.

**Comment by mxie** [comment]
A few things that are missing:
* 5 weeks in advance - set up registration
* 4 weeks in advance
  - mailing list campaign should be set up w/ scheduled release
  - coordinate goodies/freebies
* 3 weeks in advance - registration opens
* 1 week in advance
  - set up feedback survey for after the workshop
  - set up feedback docs that will be used during the workshop
* day after - send out followup email

Other stuff we've done but haven't been consistent about: having a photographer and writing recap blog posts.

File: README.md (hunk 5 above)
Comment (gabebw) [inline README.md]: Say "Registration Coordinator"?

File: README.md (hunk 3 above)
Comment (gabebw) [inline README.md]: Done.

**Comment by gabebw** [comment]
OK, applied everyone's feedback so far.

File: README.md [hunk 6]
```diff
@@ -1,56 +1,181 @@
...
+  security desk.
+* Include all attendees, coordinators, TAs, guests, and waitlist names.
+* Include the name of the event, dates, and room in the body of the email. For
+  example.:
+
+    > RailsBridge Boston event on 10/4/2013 from 7pm-10pm and 10/5/2013
+    > from 9:30am-5pm in the Mann/Sampson/Paul room
+
+* Send the list as a file attachment to nerdconf@microsoft.com, CC'ing whoever
+  our NERD Center contact is and the lead coordinators.
```
Comment (galiat) [inline README.md]: Reiterate our requested room set up (U shaped clusters) in this email.

File: README.md [hunk 7]
```diff
@@ -1,56 +1,181 @@
...
+
+Find and book a bar/restaurant for our after-party at least a week in advance.
+For more information see our [after party venues doc](/after-party-venues.md).
+
+## 1 week before (Organizer)
+
+Make sure the docs are updated and any changes to Installfest VMs are live on
+the website.
+
+## 3 days before (Organizer)
```
Comment (galiat) [inline README.md]: This could be Venue Coordinator

File: README.md [hunk 8]
```diff
@@ -1,56 +1,181 @@
...
+## Friday Night (everyone)
+
+#### TA organizer
+
+* Remind the TAs about the kind of conduct we expect and how to talk to new
+  programmers. [No feigned surprise][no-fs]!
+
+[no-fs]: https://www.recurse.com/manual#no-feigned-surprise
+
+#### MC
```
Comment (galiat) [inline README.md]: Ideally before Friday night, MC should have slide deck ready with Sponsor logos, google doc w/ link, schedule, etc. We could link to a previous slide deck here if helpful.

File: task-descriptions.md [hunk 9]
```diff
@@ -45,37 +45,15 @@ Responsibilities:
...
-* Mention after-party - how to get there, what is covered, who's sponsoring
-* Lead or encourage frequent stretching in between sections
-* Talk about next steps
-* Encourage connecting with TAs and other attendees
-
 ## Photography
+
 Time commitment: part or all of the workshop; selecting && emailing photos afterward
 
 * Take brilliant pictures.
```
Comment (galiat) [inline task-descriptions.md]: Possibly give photog permission to post to twitter during the event

File: README.md (hunk 8 above)
Comment (gabebw) [inline README.md]: Great idea. I added it as 1 week before.

File: task-descriptions.md (hunk 9 above)
Comment (gabebw) [inline task-descriptions.md]: Done.

**Pull request merged**